dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheStoreService.java

```diff
@@ -90,22 +89,25 @@ protected void activate(ComponentContext context, Map<String, Object> props) {
 
         // load JCache provider from configured library, which is either specified as a libraryRef or via a bell
         CachingProvider provider = Caching.getCachingProvider(library.getClassLoader());
-        cacheManager = provider.getCacheManager(null, new CacheClassLoader()); // TODO When class loader is specified, it isn't being used for deserialization. Why?
-        cache = cacheManager.getCache("com.ibm.ws.session.cache", SessionKey.class, SessionData.class);
+        cacheManager = provider.getCacheManager(null, null, null);
```

Comment:
could just call `provider.getCacheManager()` now

Reply:
I'm intentionally using the signature that allows for sending the properties because we will eventually need to do that.